ivy/container/extensions/creation.py

```diff
@@ -31,11 +30,10 @@ def static_triu_indices(
             prune_unapplied,
             map_sequences,
             device=device,
-            out=out,
         )
 
-    def triu_indices(
-        self: ivy.Container,
```

Comment:
We shouldn't be removing other code that isn't related to the PR, could you go over the changes and make sure no unrelated changes are made?

Reply:
Got it, I will create separate PR for it.